frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -185,6 +218,8 @@ export default function CreateBooking() {
 
   const convertLocalTime = (date: Date) => {
     const newDate = new Date(date.getTime() + 28800000)
+    console.log(newDate)
+    format(newDate, 'dd/MM/yyyy, hh:mm:ss')
```

Comment:
can discuss with @jason-ng-zq99 see if yall want to use date-fns, or self-code date formatting. Either way i think can abstract out into a utils or common file